examples/CMakeLists.txt

```diff
@@ -43,6 +43,23 @@ append_host_compiler_options(CMAKE_CXX_FLAGS "${DPCPP_CXX_NOWARN_FLAGS}")
 file(GLOB_RECURSE sources *.cpp *.c)
 file(GLOB_RECURSE headers *.hpp *.h)
 
+
+# Remove tests for CUDA which use unimplemented primitives
+if(DNNL_SYCL_CUDA)
+    message("removing items")
```

Comment:
Normally, we don't print anything if we remove some source files from build. Let's remove this message,

Reply:
This was a mistake I changed it